### PR TITLE
Bump qe-tools dependency

### PR DESCRIPTION
**Pull request by rmohr**
Pull in the newer version of the qe-tools ginkgo reporter.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -69,7 +69,7 @@ require (
 	k8s.io/utils v0.0.0-20190801114015-581e00157fb1
 	kubevirt.io/client-go v0.0.0-00010101000000-000000000000
 	kubevirt.io/containerized-data-importer v1.10.9
-	kubevirt.io/qe-tools v0.1.3-0.20190512140058-934db0579e0c
+	kubevirt.io/qe-tools v0.1.5
```
Comment (nellyc) [inline go.mod:72]: Im afraid I had a bug in this version.
I have created a new release now,
mind updating to v0.1.6?

Comment (rmohr) [reply go.mod]: Done.

**Comment by rmohr** [comment]
/override pull-kubevirt-e2e-k8s-1.15-ceph

the usual ceph test. I created an extra PR to debug that test and finally fix it.

**Comment by rmohr** [comment]
Ups wrong tab. But should not matter.

**Comment by nellyc** [comment]
where do we stand with this one? 
i would really like to benefit from the new abilities this version of qe-tools brings :)

**Comment by rmohr** [comment]
@vladikr would you mind having a look?

**Comment by vladikr** [comment]
> @vladikr would you mind having a look?

@rmohr Sure, looking

**Comment by nellyc** [comment]
@rmohr can we please cherry pick into 0.28?

**Comment by rmohr** [comment]
We have no 0.28 branch. Did you mean 0.27 or 0.29?

**Comment by nellyc** [comment]
0.29 please

**Pull request merged**